docusaurus/docs/develop/path/7_e2e_and_load_tests.md

```diff
@@ -212,23 +227,33 @@ Tests will **fail** if any configured thresholds are exceeded, ensuring consiste
 
 ### Running Binary manually (no docker)
 
-**In one shell, run:**
+Load tests may also be run against a local PATH instance.
+
+**To enable this, first set `e2e_load_test_config.load_test_config.gateway_url_override` to `http://localhost:3069/v1` in the file `./e2e/config/.e2e_load_test.config.yaml`.**
+
+**Then, in one shell, run:**
 
 ```bash
 # Replace with .morse.config.yaml for Morse
 cp ./e2e/config/.shannon.config.yaml ./local/path/.config.yaml
 make path_run
 ```
 
-In another shell, run:
+**Once the PATH instance is running, in another shell, run one of the following:**
 
 ```bash
-make test_e2e_evm_shannon GATEWAY_URL_OVERRIDE=http://localhost:3069/v1
+# Shannon load tests with default service IDs
+make test_load_evm_shannon_defaults
+
+# Shannon load tests with specified service IDs
+make test_load_evm_shannon eth,anvil
 ```
 
 ### Reviewing PATH Logs
 
-By default, the logs are written to `./path_log_e2e_test_{timestamp}.txt`.
+In E2E test mode, logs may be written to `./path_log_e2e_test_{timestamp}.txt`.
+
+**In order to enable this, set the field `e2e_load_test_config.e2e_config.docker_config.log_to_file` to `true` in the file `./e2e/config/.e2e_load_test.config.yaml`.**
```

Comment:
Ditto.

Give the user a copy-pasta `sed` command so they don't even need to think.

I'm sure AI can help put this together quickly.

Reply:
Ditto, done.